tools/util-linux/Makefile

```diff
@@ -7,11 +7,11 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=util-linux
-PKG_VERSION:=2.39.3
+PKG_VERSION:=2.40.1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=@KERNEL/linux/utils/$(PKG_NAME)/v2.39
-PKG_HASH:=40ea07584d56c310455471afa92c119ec259776a561af7159cc802344c2c370d
+PKG_SOURCE_URL:=@KERNEL/linux/utils/$(PKG_NAME)/v2.40
```

Comment:
Maybe

```
PKG_SOURCE_URL:=@KERNEL/linux/utils/$(PKG_NAME)/v$(basename $(PKG_VERSION))
```

Reply:
The non-point releases do not have a trailing `.0`, e.g `2.40.`, so that doesn't work.